### PR TITLE
Hover text

### PR DESCRIPTION
**Pull request by jdamiba**

### CODE REVIEW

**Comment by jdamiba** [comment]
This PR replaces https://github.com/plotly/documentation/pull/1595

**Comment by nicolaskruchten** [comment]
@Mahdis-z can you get this PR updated for inclusion into this repo? basically without the `api_create` calls and following the same pattern as the other files :)

**Comment by nicolaskruchten** [comment]
ah, doh, I see that @jdamiba has updated the PR already for the new pattern! nice job Joe!

@Mahdis-z then please review this PR to make sure it matches your intention from the previous one it replaces :)

**Comment by Mahdis-z** [comment]
> ah, doh, I see that @jdamiba has updated the PR already for the new pattern! nice job Joe!
> 
> @Mahdis-z then please review this PR to make sure it matches your intention from the previous one it replaces :)

sounds good to me 👍

File: r/2018-01-29-hover-text-and-formatting.Rmd [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
+name: Hover Text and Formatting
+permalink: r/hover-text-and-formatting/
 description: How to use hover text and formatting in R with Plotly.
-display_as: file_settings
-language: r
 layout: base
-name: Hover Text and Formatting
+thumbnail: thumbnail/hover'text.jpg
```
Comment (nicolaskruchten) [inline r/2018-01-29-hover-text-and-formatting.Rmd]: this doesn't seem right, the `'` character

Comment (nicolaskruchten) [reply r/2018-01-29-hover-text-and-formatting.Rmd]: I'll fix it

Comment (Mahdis-z) [reply r/2018-01-29-hover-text-and-formatting.Rmd]: should be hover-text.png

**Pull request merged**